main.cpp

```diff
@@ -100,6 +102,21 @@ int main(int argc,char *argv[])
 	limit.rlim_cur = 1024;
 	limit.rlim_max = 1024;
 
+	// Browse argv and try to find the --cfgFile option
+	int found_cfg = 0;
+	int k = 0;
+	while(k<argc && !found_cfg) {
+	  found_cfg = (strcasecmp(argv[k],"--cfgFile")==0);
```

Comment:
Perhaps should we rename the option --cfgFile (which sounds a little generic) with (i.e.) --aclFile (which sounds a more bit  specific)?

Reply:
According to POSIX it should be `--acl-file`

Reply:
I thought that be consistent was the right approach. Please let me what is for you the best name for that option!